multiqc/modules/fgbio/ErrorRateByReadPosition.py

```diff
@@ -73,33 +85,48 @@ def parse_reports(self):
         'ymin': 0,
         'data_labels': [
             {'name': 'Error Rate', 'ylab': 'Error Rate'},
-            {'name': 'A > C', 'ylab': 'A to C error rate'},
-            {'name': 'A > G', 'ylab': 'A to G error rate'},
-            {'name': 'A > T', 'ylab': 'A to T error rate'},
-            {'name': 'C > A', 'ylab': 'C to A error rate'},
-            {'name': 'C > G', 'ylab': 'C to G error rate'},
-            {'name': 'C > T', 'ylab': 'C to T error rate'},
-        ]
+            {'name': 'A > C', 'ylab': 'A to C error rate (and T to G when collapsed)'},
+            {'name': 'A > G', 'ylab': 'A to G error rate (and T to C when collapsed)'},
+            {'name': 'A > T', 'ylab': 'A to T error rate (and T to A when collapsed)'},
+            {'name': 'C > A', 'ylab': 'C to A error rate (and G to T when collapsed)'},
+            {'name': 'C > G', 'ylab': 'C to G error rate (and G to C when collapsed)'},
+            {'name': 'C > T', 'ylab': 'C to T error rate (and G to A when collapsed)'},
```

Comment:
I think the added text clutters the legend.  Can you add this information to the title or elsewhere?

Reply:
I kind of think the help text covers it anyways, so I've just removed it.